docs/community/contribute.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: layouts/community.njk
+title: Propose a contribution
+---
+
+Our contribution process supports teams to add styles, components and patterns based on their users needs and service requirements, whilst guaranteeing the quality and usefulness of the MOJ Design System.
+
+Anyone can propose a new style, component or pattern.
+
+### Step 1: Check the contribution criteria
+
+The MOJ Design System Group review contributions against a [contribution criteria](/community/criteria). To support the review of contributions, first check that your proposal meets the [criteria](/community/criteria).
+
+### Step 2: Check for similar contributions
+
+Check if someone has already suggested your idea, or something similar, by viewing a list of contributions under review in the [contribution backlog](https://docs.google.com/spreadsheets/d/1TlF5XxAoE_SPB9JBg1IktY0W30d4kMElwSLGclzL0wQ/edit?usp=sharing).
+
+### Step 3: Propose your contribution
+
+Propose a contribution using the [contribution form](https://forms.gle/Hp56F9mYxz8Mh4BZ8). When submitted, your contribution can be viewed on the [contribution backlog](https://docs.google.com/spreadsheets/d/1TlF5XxAoE_SPB9JBg1IktY0W30d4kMElwSLGclzL0wQ/edit?usp=sharing).
```

Comment:
I think we should also highlight the ability to suggest a contribution over GitHub, since that's an easier way of sharing code changes for anyone who's already familiar with it.

Reply:
Agreed, as we want contributions to work for everyone so tooling needs to be right for different people.

But right now we need to balance this with handling contributions in an organised way that we can manage and review as a the MOJ Design System Group. Not suggesting this is perfect right now, but it's low barrier for people to contribute and easy for us to review contributions as a group.

There haven't been contributions for a while, and there could be a lot of contributions soon to come in. Having them coming in in different ways will mean we'll struggle to be organised at this very early stage.